packages/dependency-extraction-webpack-plugin/package.json

```diff
@@ -16,7 +16,7 @@
 	"bugs": {
 		"url": "https://github.com/woocommerce/woocommerce-admin/issues"
 	},
-	"main": "build/index.js",
+	"main": "src/index.js",
 	"module": "build-module/index.js",
```

Comment:
We should remove the module declaration, since that won't function and would confuse someone using this package outside of node.js by accident.